### ISSUE TITLE
Download firmware - Synology

### ISSUE THREAD

**Issue by IceFlovver** [amosproj/amos2022ws01-firmware-scraper#9]
## User story
1. As a user after having chosen Synology product
2. I want the software to search the website
3. So that the download can be initiated

## Acceptance criteria
* Correct firmware is downloaded

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
